user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_diagnostic_procedures/SLNetClientTest_inspecting_active_replication_buffers.md

```diff
@@ -26,8 +26,38 @@ To drop one specific replication buffer:
 
 1. In the *Type* field at the bottom, select *ReplicationBufferStats*.
 
-> [!NOTE]
-> When a replication buffer saves files to disk, those files are located in C:\\Skyline DataMiner\\System Cache\\SLNet and are named "ReplicationBuffer_clienthostname_ip_dmaid_eid_objectid.bin".
-
 > [!WARNING]
 > Always be extremely careful when using the SLNetClientTest tool, as it can have far-reaching consequences on the functionality of your DataMiner System.
+
+## Deleting and flushing replication buffer files
+
+When a replication buffer saves files to disk, those files are located in `C:\Skyline DataMiner\System Cache\SLNet` and are named "ReplicationBuffer_clienthostname_ip_dmaid_eid_objectid.bin".
```

Comment:
As the _objectid (the unique hash) is no longer part of the name of the file, I'm wondering if this line should reflect the current state or how it used to be. In the next paragraph, it's mentioned that this is indeed dropped since version X.